src/pathfinding_service/cli.py

```diff
@@ -47,6 +47,19 @@
     help="Number of block confirmations to wait for",
 )
 @click.option("--enable-debug", default=False, is_flag=True, hidden=True)
+@click.option("--pfs-operator", default="John Doe", type=str, help="Name of the service operator")
+@click.option(
+    "--pfs-settings",
```

Comment:
I wonder if we should remove this for now, as it doesn't have a purpose right now.

Reply:
yep that is a good idea